tests/redirects/map_globalconf.py

```diff
@@ -280,9 +280,6 @@
     url_test('/thunderbird/about/{board,press,staff}/',
              'http://website-archive.mozilla.org/www.mozilla.org/thunderbird/thunderbird/about/{board,press,staff}/'),
 
-    # bug 1121082
-    url_test('/hello/', '/firefox/hello/'),
```

Comment:
I wasn't sure of standard practice on keeping URL tests when effectively killing a page, but it makes sense to keep tests for the redirect. I've restored these.